app/menu.js

```diff
@@ -53,6 +53,34 @@ var createDefaultMenu = function(app, checkForUpdates) {
           ]
         },
         {
+          label: 'File',
+          submenu: [
+            {
+              label: 'New Message',
+              accelerator: 'Command+N',
+              click: function() {
+                var mainWindow = BrowserWindow.fromId(1);
+                mainWindow.webContents.send('triggerWindowEvent', 'open-mixmax-compose');
+              }
+            },
+            {
+              label: 'Refresh',
```

Comment:
I notice that many native Mac apps use "Refresh" instead of "Reload", e.g. Front, Sublime, Tower. Also, we don't need a "View" menu, since we don't have that many options.